eng/pipelines/common/global-build-job.yml

```
@@ -25,10 +25,10 @@ jobs:
     variables: ${{ parameters.variables }}
 
     steps:
-    - ${{ if eq(parameters.osGroup, 'OSX') }}:
+    - ${{ if in(parameters.osGroup, 'OSX', 'iOS', 'tvOS') }}:
       - script: |
           $(setScriptToEchoAndFailOnNonZero) 
-          brew install autoconf automake icu4c libtool openssl pkg-config python3
+          sh $(Build.SourcesDirectory)/eng/install-native-dependencies.sh $(osGroup)
```

looks like this needs to be `parameters.osGroup` and I think we can remove the `setScriptToEchoAndFailOnNonZero` to make it consistent with the other places.
```suggestion
          sh $(Build.SourcesDirectory)/eng/install-native-dependencies.sh ${{ parameters.osGroup }}
```

---

I force-pushed 1956d5e1177e149b9d904f0c2175ae92fead7c42. But why is it different from the coreclr job? 🤯
https://github.com/dotnet/runtime/blob/89b2f425fadecd00fde47ad6a397e9065711ed61/eng/pipelines/coreclr/templates/build-job.yml#L107

---

I think that's because `osGroup` is part of a `variables` YML section there which is set to `${{ parameters.osGroup }}`.